pandas/core/internals/managers.py

```diff
@@ -1617,6 +1617,15 @@ def get_slice(self, slobj: slice, axis: int = 0) -> SingleBlockManager:
     def index(self) -> Index:
         return self.axes[0]
 
+    @property
+    def array(self) -> ArrayLike:
```

Comment:
could put this on SingleDataManager and return `self.arrays[0]`?

Reply:
Yes, moved it to the base class (note this will require some type ignoring, because `.arrays` is not exactly the same for both classes)